### PR TITLE
Feature/9342 note about qr code cov pass check app

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
Added a new notice text and info button to the QRCode view. Also added a new screen for informations about CovPass App and updates some text layouts.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9342

## Screenshots
Screen Shot 2021-09-21 at 14 47 08
Screen Shot 2021-09-21 at 16 37 13
Screen Shot 2021-09-21 at 14 56 28
Screen Shot 2021-09-21 at 14 56 20
Screen Shot 2021-09-21 at 17 46 44

### CODE REVIEW

**Comment by Ein-Tim** [comment]
@ UA-Team

Wouldn't it make more sense to have the text:

"Sie selbst können Zertifikate in der Corona-Warn-App vor Reisen auf Gültigkeit im Reiseland prüfen und benötigen dazu nicht die CovPassCheck-App"? 

Because the "Gültigkeit prüfen" feature is not described as a general validity check but rather like this: 



Just my opinion here 😅

File: src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHeadlineWithImageCell.swift [hunk 1]
```diff
@@ -43,6 +46,8 @@ class DynamicTableViewHeadlineWithImageCell: UITableViewCell {
 		backgroundImageView.contentMode = .scaleAspectFit
 		contentView.addSubview(backgroundImageView)
 
+		topInsetConstraint = headlineLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 64.0)
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHeadlineWithImageCell.swift:49]: Why is `64` set here if it gets set by `func configure(headline: String, image: UIImage, topInset: CGFloat)`?

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/CovPassCheckInformation/CovPassCheckInformationViewController.swift [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+class CovPassCheckInformationViewController: DynamicTableViewController, DismissHandling {
+
+	// MARK: - Init
+
+	init(
+		onDismiss: @escaping () -> Void
+	) {
+		self.onDismiss = onDismiss
+
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/CovPassCheckInformation/CovPassCheckInformationViewController.swift:15]:
```suggestion
```

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/CovPassCheckInformation/__tests__/CovPassCheckInformationViewModelTests.swift [hunk 3]
```diff
@@ -0,0 +1,22 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class CovPassCheckInformationViewModelTests: XCTestCase {
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/CovPassCheckInformation/__tests__/CovPassCheckInformationViewModelTests.swift:8]: At least we test something :D

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Views/HealthCertificateQRCodeView.swift [hunk 4]
```diff
@@ -71,19 +97,48 @@ class HealthCertificateQRCodeView: UIView {
...
 
+	@objc
+	private func didHitInfoButton() {
+		infoButtonaction?()
+	}
+
+	// MARK: - Unitest helpers
+
+	#if DEBUG
+	var noticrLabelIsHidden: Bool {
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Views/HealthCertificateQRCodeView.swift:134]:
```suggestion
	var noticeLabelIsHidden: Bool {
```

**Review by marcussc — APPROVED** [review]
Some small comments/Questions otherwise LGTM

File: src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHeadlineWithImageCell.swift (hunk 1 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHeadlineWithImageCell.swift:49]: good point, I removed this old default value

**Review by naveeddotio — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**